spark/src/main/java/org/apache/iceberg/spark/source/SparkWriteBuilder.java

```diff
@@ -164,10 +164,10 @@ private void validatePartitionTransforms(PartitionSpec spec) {
     }
   }
 
-  private boolean checkNullability(CaseInsensitiveStringMap options) {
+  private boolean checkNullability(CaseInsensitiveStringMap opts) {
```

Comment:
any purpose of this change?

Reply:
There is a member variable named options, the style checker reports "hidden variable" things if we use the same variable name here.

Reply:
I see.